openwrt/files-mt7622/etc/config/network.0

```diff
@@ -0,0 +1,111 @@
+config interface 'loopback'
```

Comment:
Im opting to embed the configs for the network and the wireless vs template them. I dont think contorting the existing config is worth it given how static this typically is. Open to ideas here

Reply:
I really don't understand the need for a ula_prefix in OpenWRT. If we can avoid it, I'd rather do so. We have no use for ULA in our environment. OTOH, it seems that OpenWRT may perhaps break badly without one for reasons utterly passing understanding. I haven't had time to dig into this and/or rant about it upstream, but if you can look at that and provide some feedback, that would be appreciated.

Reply:
thanks for the catch @owendelong This was just a carrier over from the default auto gen config from openwrt. Removing it doesnt break anything from my testing.

It comes from the uci defaults: https://github.com/openwrt/openwrt/blob/master/package/base-files/files/etc/uci-defaults/12_network-generate-ula but why thats a default I dont know either.